FirebaseAppDistribution/Sources/FIRAppDistributionUIService.h

```diff
@@ -44,6 +53,9 @@ typedef void (^AppDistributionRegistrationFlowCompletion)(NSError *_Nullable err
 
 - (void)showUIAlert:(UIAlertController *)alertController;
 
```

Comment:
This could be potentially simplified to use a single completion handler. See comment above.